client/src/pages/Profile/Profile.jsx

```diff
@@ -826,7 +826,9 @@ const ProfilePageHeader = ({ leader, editButton }) => {
               <img src={EditIcon} alt={'edit'} className={'profile-edit-icon'} />
             </Link>
           ) : (
-            <></>
+            <Link to={'/profile-edit-unregistered'} className={'profile-edit-icon-link no-link-style'}>
+              <img src={EditIcon} alt={'edit'} className={'profile-edit-icon'} />
+            </Link>
```

Comment:
```jsx
{editButton !== false && (
        <Link to={isRegistered? '/profile-edit' : '/profile-edit-unregistered'} className={'profile-edit-icon-link no-link-style'}>
          <img src={EditIcon} alt={'edit'} className={'profile-edit-icon'} />
        </Link>
)}
```
Use this instead since the original removes the functionality of the `editButton` variable by returning the button in both conditions even if `editButton` is false.